Library/Formula/weechat.rb

```diff
@@ -30,12 +34,13 @@ def install
 
     args = std_cmake_args
 
-    args << "-DENABLE_LUA=OFF"    if build.without? "lua"
-    args << "-DENABLE_PERL=OFF"   if build.without? "perl"
-    args << "-DENABLE_RUBY=OFF"   if build.without? "ruby"
-    args << "-DENABLE_ASPELL=OFF" if build.without? "aspell"
-    args << "-DENABLE_GUILE=OFF"  if build.without? "guile"
-    args << "-DENABLE_PYTHON=OFF" if build.without? "python"
+    args << "-DENABLE_LUA=OFF"        if build.without? "lua"
```

Comment:
We don't really care about keeping lines the exact same length, and generally frown on unnecessary whitespacing which is why I'm a bit :-1: on it.